print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
+
+  printList() {
+    let current = this;
+
+    while (current != null) {
+      console.log(current.value);
+      current = current.getNext();
+    }
```

Comment:
I think I'd prefer to see this build up something like "1 -> 2 -> 3 -> null".

Reply:
I agree, Kaida. Good suggestion.

Reply:
Sounds good!